src/axom/quest/examples/quest_marching_cubes_example.cpp

```diff
@@ -0,0 +1,1491 @@
+    axom::Array<axom::StackArray<axom::IndexType, DIM>> domainLengths(domainCount);
+    for(axom::IndexType n = 0; n < domainCount; ++n)
+    {
+      auto& domain = computationalMesh.domain(n);
+      axom::ArrayView<const double, DIM>* domainCoordsView =
+        &coordsViews[DIM * n];
+      get_coords_views(domain, computationalMesh.coordset_path(), domainCoordsView);
+
+      axom::Array<axom::IndexType> domLengths =
+        computationalMesh.domain_lengths(n);
+      for(int d = 0; d < DIM; ++d)
+      {
+        domainLengths[n][d] = domLengths[d];
+      }
+    }
+
+    for(axom::IndexType cn = 0; cn < cellCount; ++cn)
+    {
+      axom::IndexType domainId = domainIds[cn];
+      const axom::StackArray<axom::IndexType, DIM>& domainSize =
+        domainLengths[domainId];
+      // conduit::Node& domain = computationalMesh.as_conduit_node()[domainId];
+
+      axom::IndexType parentCellId = parentCellIds[cn];
+      axom::StackArray<axom::IndexType, DIM> parentCellIdx =
+        flat_to_multidim_index(parentCellId, domainSize);
+      reverse(parentCellIdx);  // ArrayView expects indices in reverse order.
+      axom::StackArray<axom::IndexType, DIM> upperIdx = parentCellIdx + 1;
+
+      axom::ArrayView<const double, DIM>* domainCoordsView =
+        &coordsViews[DIM * domainId];
+      axom::primal::Point<double, DIM> lower, upper;
+      for(int d = 0; d < DIM; ++d)
+      {
+        lower[d] = domainCoordsView[d][parentCellIdx];
+        upper[d] = domainCoordsView[d][upperIdx];
+      }
+      axom::primal::BoundingBox<double, DIM> parentCellBox(lower, upper);
+      double tol = error_tolerance();
+      axom::primal::BoundingBox<double, DIM> big(parentCellBox);
+      axom::primal::BoundingBox<double, DIM> small(parentCellBox);
+      big.expand(tol);
+      small.expand(-tol);
+
+      // WRONG: the node ids should increased by the number of nodes in all previous domains.
+      axom::IndexType* cellNodeIds = contourMesh.getCellNodeIDs(cn);
+      const axom::IndexType cellNodeCount = contourMesh.getNumberOfCellNodes(cn);
+
+      for(axom::IndexType nn = 0; nn < cellNodeCount; ++nn)
+      {
+        axom::primal::Point<double, DIM> nodeCoords;
+        contourMesh.getNode(cellNodeIds[nn], nodeCoords.data());
+
+        if(!big.contains(nodeCoords) || small.contains(nodeCoords))
+        {
+          ++errCount;
+          SLIC_INFO_IF(
+            params.isVerbose(),
+            axom::fmt::format("check_surface_cell_limits: node {} at {} is not "
+                              "on parent cell boundary.",
+                              cellNodeIds[nn],
+                              nodeCoords));
+        }
+      }
+    }
+
+    SLIC_INFO_IF(params.isVerbose(),
+                 axom::fmt::format("check_surface_cell_limits: found {} nodes "
+                                   "not on parent cell boundary.",
+                                   errCount));
+    return errCount;
+  }
+
+#if 1
```

Comment:
Please remove `#if 1` and its corresponding `#endif`.  I can understand the `#if 0` at line 883 and 509 preserving code for particularly thorny debugging sessions, but I don't see the need to keep this line and line 1148.